### ISSUE TITLE
Error due to no module dependencies 

### ISSUE THREAD

**Issue by ldez** [kropp/intellij-makefile#111]
I use GoLand (the JetBrains IDE for Golang) and I have the following error:

> Plugin Error: Plugin "Makefile support" defines no module dependencies (supported only in IntelliJ IDEA).

```
GoLand 2019.3.1
Build #GO-193.5662.65, built on December 23, 2019

Runtime version: 11.0.5+10-b520.17 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.19.96-1-MANJARO
```

Is it expected?

**Comment by MrSquaare** [comment]
Same problem, the plugin seems to be supported only in IntelliJ IDEA.

```
Plugin Error: Plugin "Makefile support" defines no module dependencies (supported only in IntelliJ IDEA).
```

**Comment by avalentino** [comment]
Same here:
```
PyCharm 2019.3.1 (Community Edition)
Build #PC-193.5662.61, built on December 18, 2019

Runtime version: 11.0.5+10-b520.17 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 5.3.0-28-generic
```

**Comment by rebrendov** [comment]
Have a similar issue. Tried to install v1.7 and it works without the problem.
```
GoLand 2019.3.1
Build #GO-193.5662.65, built on December 23, 2019

Runtime version: 11.0.5+10-b520.17 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 5.0.0-38-generic
```

**Comment by Komzpa** [comment]
Same issue in DataGrip:
![image](https://user-images.githubusercontent.com/810638/72678023-2a369a80-3ab3-11ea-9255-2ff4374e9b93.png)

**Comment by andisolo** [comment]
same issue in clion.

**Comment by wtw24** [comment]
Same issue in PhpStorm:

PhpStorm 2019.3.1
Build #PS-193.5662.63, built on December 19, 2019

Runtime version: 11.0.5+10-b520.17 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 5.3.0-26-generic

Registry: run.processes.with.pty=TRUE
Non-Bundled Plugins: name.kropp.intellij.makefile, org.jetbrains.plugins.rest

**Comment by ldez** [comment]
The problem appears to have been corrected 471c03da8b1f76f0370f677c4bec4cdf5100461c

We have to wait for the publication of the v2.0.1 version.

**Comment by kropp** [comment]
Fixed in 2.0.1
